.travis.yml

```diff
@@ -88,10 +88,11 @@ install:
 script:
   - |
     if [ -z "$STACK_YAML" ]; then
-       cabal new-test
+      cabal new-test
     else
       stack build --test --no-terminal
     fi
+  - curl -sSL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s .
```

Comment:
@qoelet Thanks for your contribution! Having `hlint` check on CI is really good. Though, we can improve the configuration even better. `summoner` uses `relude` package as an alternative prelude, which means that we can use HLint rules from `relude`:

* https://github.com/kowainik/relude/blob/master/.hlint.yaml

I would appreciate that in Travis CI we download `.hlint.yaml` file for the version of `relude` we're using, rename this file to `.hlint-relude.yaml` (just in case we will have default `.hlint.yaml` in future) and run `hlint` with `.hlint-relude.yaml` hints.

Reply:
@ChShersh updated the PR, let me know if that's OK.